emr/application/views/screeners/add.php

```diff
@@ -106,12 +106,12 @@
 							</div>	
 							<div class="col-xs-6">
 								<ul class="nav navbar-right panel_toolbox">
-									<li><button type="button" class="btn btn-danger" onclick=history.back()>Back</button></li>
+									<li><button type="button" class="btn btn-danger" onclick=history.back() style="padding: 5px 18px 5px 18px; margin: 5px 0 5px 5px;">Back</button></li>
```

Comment:
you shouldn't do history.back(), and you should have a href instead

Reply:
Changed

Reply:
refer to my new comment on your changed code